nautobot/dcim/utils.py

```diff
@@ -49,6 +61,62 @@ def cable_status_color_css(record):
     return f"background-color: #{lighter_color}"
 
 
+def get_network_driver_mapping_tool_names():
+    """
+    Return a list of all available network driver tool names derived from the netutils library and the optional NETWORK_DRIVERS setting.
+
+    Tool names are "ansible", "hier_config", "napalm", "netmiko", etc...
+    """
+    network_driver_names = NETUTILS_NETWORK_DRIVER_MAPPING_NAMES.copy()
+    network_driver_names.update(get_settings_or_config("NETWORK_DRIVERS").keys())
+
+    return sorted(network_driver_names)
+
+
+def get_network_driver_mappings():
```

Comment:
nit, non-blocking -- perhaps `get_all_network_driver_mappings` might be clearer?